tools/mo/requirements.txt

```diff
@@ -4,7 +4,7 @@ mxnet~=1.2.0; sys_platform == 'win32'
 mxnet>=1.7.0.post2,<=1.9.1; sys_platform != 'win32'
```

Comment:
BTW, why do we name TF, mxnet and ONNX in a list of common MO requirements? as well as we have dedicated files like https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/requirements_mxnet.txt

Reply:
I will clean-up this requirements file so we will have only numpy, defusedxml, networkx dependencies.
Sure these fws should not be in this file